modules/olm-updating-index-image.adoc

```diff
@@ -34,21 +34,40 @@ endif::[]
 [source,terminal]
 ----
 $ opm index add \
-    --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
-    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
-    --pull-tool podman <4>
+    --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \
+    --from-index <registry>/<namespace>/<existing_index_image>:<existing_tag> \
+    --tag <registry>/<namespace>/<existing_index_image>:<updated_tag> \
+    --pull-tool podman
+----
++
+where:
++
+[small]
+--
+<registry>:: Specifies the hostname of the registry, such as `quay.io` or `mirror.example.com`.
+<namespace>:: Specifies the namespace of the registry, such as `ocs-dev` or `abc`.
+<new_bundle_image>:: Specifies the new bundle image you want to add to the registry, such as `ocs-operator`.
+<digest>:: Specifies a comma-separated list of additional bundle images to add to the index, such as `c7f11097a628f092d8bad148406aa0e0951094a03445fd4bc0775431ef683a41`.
```

Comment:
It's confusing here. The `digest` belongs to the image, it cannot be `comma-separated`. The `--bundles` specifies a comma-separated list of additional bundle images to add to the index, not the `digest`.

Reply:
Thank you for the feedback, @jianzhangbjz.

Would you PTAL at the changes and let me know what you think?

Reply:
Looks good to me, thanks!